test_scripts/Polices/Policy_Table_Update/026_ATF_P_TC_PTU_Merge_Into_LPT.lua

```diff
@@ -59,7 +60,31 @@ local function is_table_equal(t1, t2)
   return true
 end
 
+local function update_preloaded_pt_removeRC()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
```

Comment:
@istoimenova, It's better to move out back up of sdl_preloaded_pt.json file from the function